challenge/src/test/java/Exercise1Test.java

```diff
@@ -0,0 +1,34 @@
+import e1.Exercise1;
+import e1.Exercise1Impl;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Exercise1Test extends Exercise1Impl {
+    Exercise1 evenOrOdd = new Exercise1Impl();
+
+    @Test
+    void testIfValueIs_EvenOrOdd() {
+
+        assertEquals("Even", evenOrOdd.verify(2));
```

Comment:
Os cenários que me parecem válidos são esses.
```suggestion
                assertEquals("Even", evenOrOdd.verify(2));
        assertEquals("Even", evenOrOdd.verify(4));
        assertEquals("Even", evenOrOdd.verify(-2));
        assertEquals("Even", evenOrOdd.verify(0));
        assertEquals("Odd", evenOrOdd.verify(5));
        assertEquals("Odd", evenOrOdd.verify(7));
        assertEquals("Odd", evenOrOdd.verify(-1));
        assertEquals("Odd", evenOrOdd.verify(-3));
```

Veja oq acha e veja se os testes passem.

Reply:
Os testes das exceções de numero negativo, não deveriam ser separados do testIfValueIs_EvenOrOdd();?

Reply:
```suggestion
           @Test
    void testIfValueIs_EvenOrOdd() {
        assertEquals("Even", evenOrOdd.verify(4));
        assertEquals("Even", evenOrOdd.verify(0));
        assertEquals("Odd", evenOrOdd.verify(5));
        assertEquals("Odd", evenOrOdd.verify(7));
    }

    @Test
    void testIfValueIsNull() {
        assertThrows(NullPointerException.class, () -> evenOrOdd.verify(null));
    }

    @Test
    void testIfValueIsNegative() {
        assertThrows(IllegalArgumentException.class, () -> evenOrOdd.verify(-2));
        assertThrows(IllegalArgumentException.class, () -> evenOrOdd.verify(-1));
        assertThrows(IllegalArgumentException.class, () -> evenOrOdd.verify(-3));
    }
   
```
Achei para o valor negativo esse Exception...
Não seria assim? um teste cada exceção ?